src/development/LocalTest/appsettings.json

```diff
@@ -36,7 +36,7 @@
   "AllowedHosts": "*",
   "LocalPlatformSettings": {
     "LocalTestingStorageBasePath": "c:/AltinnPlatformLocal/",
-    "AppRepsitoryBasePath": "<Path to your app>/App/",
+    "AppRepsitoryBasePath": "<Path to your repos folder>",
```

Comment:
we could probably specify this to mean app repos folder, so it is not misunderstood.

Reply:
done